Gopkg.toml

```diff
@@ -51,7 +51,7 @@ required = [
 [[constraint]]
   # Also defined in bazelbuild/rules_go
   # https://github.com/bazelbuild/rules_go/blob/436452edc29a2f1e0edc22d180fbb57c27e6d0af/go/private/repositories.bzl#L131
-  revision = "86e600f69ee4704c6efbf6a2a40a5c10700e76c2"
+  revision = "383e8b2c3b9e36c4076b235b32537292176bae20"
```

Comment:
Will this change require a corresponding change to `bazelbuild/rules_go`?

Reply:
I see you already managed to confirm we will need this change.

Reply:
Can you update this comment as well since this is no longer correct?

Reply:
This comment still needs updating

Reply:
Done!